ocpmodels/common/utils.py

```diff
@@ -1121,3 +1136,167 @@ def scatter_det(*args, **kwargs):
         torch.use_deterministic_algorithms(mode=False)
 
     return out
+
+
+def get_commit_hash():
+    try:
+        commit_hash = (
+            subprocess.check_output(
+                ["git", "-C", ocpmodels.__path__[0], "describe", "--always"]
+            )
+            .strip()
+            .decode("ascii")
+        )
+    # catch instances where code is not being run from a git repo
+    except Exception:
+        commit_hash = None
+
+    return commit_hash
+
+
+def cg_decomp_mat(l, device="cpu"):
+    if l not in [2]:
+        raise NotImplementedError
+
+    if l == 2:
+        change_mat = torch.tensor(
+            [
+                [3 ** (-0.5), 0, 0, 0, 3 ** (-0.5), 0, 0, 0, 3 ** (-0.5)],
+                [0, 0, 0, 0, 0, 2 ** (-0.5), 0, -(2 ** (-0.5)), 0],
+                [0, 0, -(2 ** (-0.5)), 0, 0, 0, 2 ** (-0.5), 0, 0],
+                [0, 2 ** (-0.5), 0, -(2 ** (-0.5)), 0, 0, 0, 0, 0],
+                [0, 0, 0.5**0.5, 0, 0, 0, 0.5**0.5, 0, 0],
+                [0, 2 ** (-0.5), 0, 2 ** (-0.5), 0, 0, 0, 0, 0],
+                [
+                    -(6 ** (-0.5)),
+                    0,
+                    0,
+                    0,
+                    2 * 6 ** (-0.5),
+                    0,
+                    0,
+                    0,
+                    -(6 ** (-0.5)),
+                ],
+                [0, 0, 0, 0, 0, 2 ** (-0.5), 0, 2 ** (-0.5), 0],
+                [-(2 ** (-0.5)), 0, 0, 0, 0, 0, 0, 0, 2 ** (-0.5)],
+            ],
+            device=device,
+        ).detach()
+
+    return change_mat
+
+
+def irreps_sum(l):
+    total = 0
+    for i in range(l + 1):
+        total += 2 * i + 1
+
+    return total
+
+
+def load_old_config(name, config):
```

Comment:
It's not obvious to me where name comes from or should be guessed from an old config (I think it's implicit in the trainer type?). It would be cleaner if we didn't have to pass in the name here!